problem-solver/cxx/messageReplyModule/generator/MessageHistoryGenerator.cpp

```diff
@@ -25,7 +26,7 @@ void MessageHistoryGenerator::addMessageToDialog(const ScAddr & dialogAddr, cons
       ScType::EdgeAccessConstPosPerm,
       ScType::NodeConst,
       ScType::EdgeAccessConstPosPerm,
```

Comment:
```suggestion
      ScType::EdgeAccessConstPosTemp,
```
Check all arcs to be temporal
![image](https://github.com/ostis-apps/nika/assets/36566977/45908292-0d68-48ae-9165-91a9ff0642ec)

Reply:
Remove duplicated generator in message reply module